script/playbackstrategy/msestrategy.js

```diff
@@ -445,6 +479,8 @@ define('bigscreenplayer/playbackstrategy/msestrategy',
           var seekToTime = getClampedTime(time, getSeekableRange());
           if (windowType === WindowTypes.SLIDING) {
             mediaElement.currentTime = (seekToTime + timeCorrection);
+          } else if (windowType === WindowTypes.GROWING && seekToTime > getCurrentTime()) {
+            refreshManifestBeforeSeek(seekToTime);
```

Comment:
Maybe naming should be more explicit? This also carries out the seek after the manifest has been loaded maybe `refreshManifestAndSeek(...)`?

Reply:
I think I'm happy with the original!